src/dashboard/app.py

```
@@ -176,6 +176,52 @@ def get_overview_stats():
+        }), 500
+
+@app.route('/api/config/keywords', methods=['POST'])
+def update_configured_keywords():
+    """Update configured keywords for monitoring"""
+    try:
+        data = request.get_json()
+        keywords = data.get('keywords', [])
+        keywords = list(set([kw.strip() for kw in keywords if kw.strip()]))
+
+        if len(keywords) == 0:
+            raise ValueError("Keyword list cannot be empty")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)